### DISCUSSION TITLE
Using Divi Theme Using [tsml_ui] resulting in meeting list fields overlapping.

### DISCUSSION THREAD

**Discussion by ghost** [code4recovery/12-step-meeting-list#1457] [Q&A] [answered]
Hi,

     I'm using divi to setup a dedicated page for meeting listing, as it help manage the display width.

     When testing using phone sized display the 'meetings' fields are not overflowing resulting in overlapping.

     Working max width.

![image](https://github.com/code4recovery/12-step-meeting-list/assets/160238568/36d09b53-d687-4dce-a2a7-b295bb86f477)

      Overlapping.

![image](https://github.com/code4recovery/12-step-meeting-list/assets/160238568/8c49ffb2-0f97-429e-9d82-8a9509f211e9)

**Comment by joshreisner** [comment, marked as answer]
interesting that this is not an issue on https://staging17.aa-bristol.org/meetings/ - there i see that the alignment is weird, i bet you could add `#tsml-ui { margin: 0 auto; }` to your custom CSS to get it to center-align - i will also take a note to make that our margin default

looks like in the second case our padding is getting overridden by style coming from the divi event calendar module, we can make it more resilient

overall, i'd recommend using the main listing page rather than the shortcode - that is usually subject to less interference from themes and plugins

**Reply by ghost** [reply]
Thanks for taking the time. Josh
